include/ck_tile/ops/reduce/pipeline/reduce2d_shape.hpp

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2024, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck_tile/core.hpp"
+
+namespace ck_tile {
+
+template <typename BlockWarps, // num warps along seq<M, N>
+          typename BlockTile,  // block size, seq<M, N>
+          typename WarpTile,   // warp size, seq<M, N>
+          typename Vector>     // contiguous pixels(vector size) along seq<M, N>
```

Comment:
Vector would suggest that we do vectorized access on both dims M & N with provided values.
```suggestion
          typename ThreadTile>     // contiguous pixels(vector size) along seq<M, N>
```